modules/statics/src/coins.ts

```diff
@@ -8817,6 +8825,14 @@ export const coins = CoinMap.fromCoins([
     '0xe50365f5d679cb98a1dd62d6f6e58e59321bcddf',
     UnderlyingAsset.LA
   ),
+  erc20(
+    '99b6d10c-74c9-41dd-8a98-0cc35f8e006a',
+    'ladys',
+    'Milady Meme Coin',
```

Comment:
```suggestion
    'Milady',
```
Etherscan only shows the first word?

Reply:
Yes, Etherscan only shows "Milady", I used the name shared on the ticket. I will ask to the ticket creator to change.